selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/RolloutUpdateStrategyWithEditorTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.hotupdate;
+
+import static org.eclipse.che.selenium.core.project.ProjectTemplates.MAVEN_SPRING;
+
+import com.google.inject.Inject;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.eclipse.che.api.system.shared.SystemStatus;
+import org.eclipse.che.commons.lang.NameGenerator;
+import org.eclipse.che.selenium.core.client.CheTestSystemClient;
+import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
+import org.eclipse.che.selenium.core.executor.OpenShiftCliCommandExecutor;
+import org.eclipse.che.selenium.core.webdriver.WebDriverWaitFactory;
+import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.pageobject.CodenvyEditor;
+import org.eclipse.che.selenium.pageobject.Consoles;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.Menu;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.Wizard;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class RolloutUpdateStrategyWithEditorTest {
+  private static final String NAME_OF_ARTIFACT = NameGenerator.generate("quickStart", 4);
+  private static final String PROJECT_NAME = "default-spring-project";
+  private static final String COMMAND_FOR_GETTING_CURRENT_DEPLOYMENT_CHE =
```

Comment:
We can reuse the constant `COMMAND_FOR_GETTING_CURRENT_DEPLOYMENT_CHE` in the **RolloutUpdateStrategyWithWorkspacesStartStopTest** class as well.